packages/dotcom-ui-header/src/components/drawer/topLevelPartials.tsx

```diff
@@ -33,13 +33,13 @@ const Drawer = (props: THeaderProps) => {
         <DrawerTools {...editions} />
         {!props.userIsSubscribed && subscribeAction && <SubscribeButton {...subscribeAction} />}
         <Search />
-        <nav className="o-header__drawer-menu o-header__drawer-menu--primary o-header__drawer-menu--border">
+        <nav className="o-header__drawer-menu" aria-label="Edition switcher">
           {editions && <EditionsSwitcher {...editions} />}
-          <ul data-component="drawer-menu--primary__drawer-menu-list" className="o-header__drawer-menu-list">
-            {primary ? <SectionPrimary {...primary} /> : null}
-            {secondary ? <SectionSecondary {...secondary} /> : null}
-            {tertiary ? <SectionTertiary {...tertiary} /> : null}
-          </ul>
+        </nav>
+        <nav className="o-header__drawer-menu o-header__drawer-menu--primary o-header__drawer-menu--border">
```

Comment:
`o-header__drawer-menu--border` what does this class do?

Reply:
Good question! Looks like it could have been featured in an older version of `o-header`. I will remove it 👍

Reply:
Looks like there never was a `o-header__drawer-menu--border` in `o-header`. I imagine there was some custom use for it some time ago but it no longer exists.

Reply:
This has been removed 🙌